bigquery_etl/schema/__init__.py

```diff
@@ -282,3 +283,14 @@ def to_json_file(self, json_path: Path):
     def to_json(self):
         """Return the schema data as JSON."""
         return json.dumps(self.schema)
+
+    def to_bigquery_schema(self) -> List[SchemaField]:
+        """Get the BigQuery representation of the schema."""
+        with NamedTemporaryFile(suffix=".json") as f:
+            self.to_json_file(Path(f.name))
+            return [SchemaField.from_api_repr(field) for field in json.load(f)]
```

Comment:
I see this is just moving/de-duping the code formerly in `query.py`, but it seems like the temp file isn't necessary.
```suggestion
        return [SchemaField.from_api_repr(field) for field in self.schema["fields"]]
```

Reply:
This works yeah